<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MCA Settings</string>
  </property>
  <widget class="QGroupBox" name="fraMCAmode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Acquisition mode</string>
   </property>
   <widget class="QLabel" name="lblMode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPHA">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PHA</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbMCS">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MCS</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fraMCSPreset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Dwell time</string>
   </property>
   <widget class="QLabel" name="lblSeconds">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Seconds:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDwellTime">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fraMCSSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>MCS settings</string>
   </property>
   <widget class="QLabel" name="lblMCSChannels">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Channels</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbMCSChannels">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="nmrMCSLLD">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="nmrMCSULD">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblMCSLLD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>LLD</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMCSULD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ULD</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fraPHAPreset">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Acquisition preset</string>
   </property>
   <widget class="QCheckBox" name="chkPresetTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPresetTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbPresetTime">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPresetCounts">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Counts</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPresetCounts">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblLowerChannel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower channel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nmrPresetCountsLLD">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblUpperChannel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Upper channel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nmrPresetCountsULD">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fraAmplifier">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>301</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Amplifier</string>
   </property>
   <widget class="QComboBox" name="cmbCoarseGain">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblCoarseGain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Coarse gain:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbDigitalGain">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblDigitalGain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Digital gain</string>
    </property>
   </widget>
   <widget class="QSlider" name="sldrFineGain">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbNegative">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Negative</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInputPolarity">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input polarity</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbPositive">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Positive</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFineGain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fine gain:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFineGainVal">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fraHV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>High Voltage</string>
   </property>
   <widget class="QLabel" name="lblVolts">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Volts</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkHVON">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="nmrVolt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fraPHASettings">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>PHA settings</string>
   </property>
   <widget class="QComboBox" name="cmbPHAChannels">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblLLD">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>LLD</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblULD">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ULD</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblChannels">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Channels</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nmrLLD">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="nmrULD">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="fraFilter">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>281</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Filter</string>
   </property>
   <widget class="QSpinBox" name="nmrThreshold">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblThreshold">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbRT">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblRisetime">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Rise time</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbFT">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblFlatTop">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Flat top</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbPZ">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPZ">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Poles/Zero</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkBLR">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPUR">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBLR">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Digital BLR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPUR">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pile-up reject</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdAdvanced">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Advanced...</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cmdApply">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdStore">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Store</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdClose">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>460</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>rbPHA</tabstop>
  <tabstop>rbMCS</tabstop>
  <tabstop>txtDwellTime</tabstop>
  <tabstop>cmbMCSChannels</tabstop>
  <tabstop>nmrMCSLLD</tabstop>
  <tabstop>nmrMCSULD</tabstop>
  <tabstop>cmbCoarseGain</tabstop>
  <tabstop>sldrFineGain</tabstop>
  <tabstop>cmbDigitalGain</tabstop>
  <tabstop>rbPositive</tabstop>
  <tabstop>rbNegative</tabstop>
  <tabstop>chkHVON</tabstop>
  <tabstop>chkPresetTime</tabstop>
  <tabstop>txtPresetTime</tabstop>
  <tabstop>cmbPresetTime</tabstop>
  <tabstop>chkPresetCounts</tabstop>
  <tabstop>txtPresetCounts</tabstop>
  <tabstop>nmrPresetCountsLLD</tabstop>
  <tabstop>nmrPresetCountsULD</tabstop>
  <tabstop>cmbPHAChannels</tabstop>
  <tabstop>nmrLLD</tabstop>
  <tabstop>nmrULD</tabstop>
  <tabstop>nmrThreshold</tabstop>
  <tabstop>cmbRT</tabstop>
  <tabstop>cmbFT</tabstop>
  <tabstop>cmbPZ</tabstop>
  <tabstop>chkBLR</tabstop>
  <tabstop>chkPUR</tabstop>
  <tabstop>cmdAdvanced</tabstop>
  <tabstop>cmdApply</tabstop>
  <tabstop>cmdStore</tabstop>
  <tabstop>cmdClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
